sql/parser/builtins.go

```diff
@@ -443,6 +442,42 @@ var builtins = map[string][]builtin{
 	"max": aggregateImpls(boolType, intType, floatType, stringType),
 	"min": aggregateImpls(boolType, intType, floatType, stringType),
 	"sum": aggregateImpls(intType, floatType),
+
+	// Math functions
+	"abs": {
+		builtin{
+			returnType: DummyFloat,
+			types:      typeList{floatType},
+			fn: func(args DTuple) (Datum, error) {
+				return DFloat(math.Abs(float64(args[0].(DFloat)))), nil
+			},
+		},
+		builtin{
+			returnType: DummyInt,
+			types:      typeList{intType},
+			fn: func(args DTuple) (Datum, error) {
+				return abs(args[0].(DInt))
+			},
+		},
+	},
+	"pi": {
+		builtin{
+			returnType: DummyFloat,
+			types:      nil,
```

Comment:
(can you add a comment to the `builtin` struct to explain that? probably would have saved you a minute :)